static/dtale/DataViewer.jsx

```diff
@@ -219,6 +219,8 @@ class ReactDataViewer extends React.Component {
     return (
       <GridEventHandler propagateState={this.propagateState} gridState={this.state}>
         <DtaleHotkeys propagateState={this.propagateState} {...this.state} />
+        <MenuTooltip />
```

Comment:
Would it work to keep the `MenuTooltip` within `DataViewerMenu` or does it need to be at the outer most level?  Just curious

Reply:
It does work in `DataViewerMenu`, nice! That makes a lot more sense, so I'll change that. I wasn't sure if it needed to be outside the container with the `overflow: scroll`